linked_list.py

```diff
@@ -0,0 +1,60 @@
+
+#import numpy as np
+
+# insertAt
+# deleteAt
+# first
+# last
+
+class Node:
+    def __init__(self, data=None):
+        self.next = None
+        self.data = data
+
+class LinkedList:
+    def __init__(self):
+        self.tail = None
+        self.head = None
+        self.size = 0
+
+    def append(self, data):
+        if self.size == 0:
+            self.head = Node(data)
+            self.tail = self.head
+        else:
+            node = Node(data)
+            self.tail.next = node
+            self.tail = self.tail.next
+        self.size += 1
+
+    def delete(self):
+        node = self.head
+        while node is not None:
+            if node.next==self.tail:
+                node.next = None
+                self.tail = node
+            node = node.next
+        self.size -= 1
+
+    def print_items(self):
```

Comment:
Remove all print / debug from code reviews. We need a new file such as LinkedListTests.py. 

This test case file will have multiple test functions that test & assert your code. Testing for bugs through printing happens only in academia. 

Pls implement something like this & send a CR; 

https://docs.python.org/3/library/unittest.html
https://www.youtube.com/watch?v=6tNS--WetLI

LinkedListTests.py 

def testAppendOneItem()
//Create LinkedList object 
//Append one item()
//Get the item & verify if its the item you added
//Verify the size is one 

def testAppendMultipleItems()
//Create LinkedList object 
//Append multiple items in a loop()
//Get the items & verify items are the ones you added
//Verify the size of elements you added

// Think & and test cases for other APIs you implemented

Reply:
Done